src-docs/src/views/code/code_example.js

```diff
@@ -108,6 +116,30 @@ export const CodeExample = {
       demo: <CodeBlock />,
       playground: codeBlockConfig,
     },
+    {
+      title: 'Code block virtualization',
+      source: [
+        {
+          type: GuideSectionTypes.JS,
+          code: codeBlockVirtualizedSource,
+        },
+        {
+          type: GuideSectionTypes.HTML,
+          code: codeBlockVirtualizedHtml,
+        },
+      ],
+      text: (
+        <p>
+          For large blocks of code, set <EuiCode>isVirtualized</EuiCode> to
```

Comment:
```suggestion
          For large blocks of code, add <EuiCode>isVirtualized</EuiCode> to
```

When I read `set` I expect there to also be a value. But since it's a boolean and all they had to do is `add` it, this seems to read better to me.